cypress/e2e/finance/fiscalYears/rollover-fiscal-year.cy.js

```diff
@@ -0,0 +1,165 @@
+    periodEnd: `${DateTools.getCurrentDateForFiscalYear()}T00:00:00.000+00:00`,
+    description: `This is fiscal year created by E2E test automation script_${getRandomPostfix()}`,
+    series: 'FY',
+    periodStart: `${DateTools.getCurrentDateForFiscalYear()}T00:00:00.000+00:00`,
+    periodEnd: `${DateTools.getDayAfterTomorrowDateForFiscalYear()}T00:00:00.000+00:00`,
+};
+  const defaultLedger = { ...Ledgers.defaultUiLedger };
+  const firstFund = { ...Funds.defaultUiFund };
+  const secondFund = {
+    name: `autotest_fund2_${getRandomPostfix()}`,
+    code: getRandomPostfix(),
+    externalAccountNo: getRandomPostfix(),
+    fundStatus: 'Active',
+    description: `This is fund created by E2E test automation script_${getRandomPostfix()}`,
+  };
+  const firstOrder = { ...NewOrder.defaultOneTimeOrder ,
+    orderType: "Ongoing",
+    reEncumber: true,
+};
+  const secondOrder = { 
+    orderType: "Ongoing",
+    vendor: '',
+    reEncumber: true,
+   };
+  const organization = { ...NewOrganization.defaultUiOrganizations };
+  const firstOrderLineTitle = `autotest_POL_title_${getRandomPostfix()}`;
+  const secondOrderLineTitle = `autotest_POL_title_${getRandomPostfix()}`;
+  const allocatedQuantity = '1000';
+  let user;
+
+  before(() => {
+
+    cy.getAdminToken();
+    // create first Fiscal Year and prepere 2 Funds for Rollover
+    FiscalYears.createViaApi(firstFiscalYear)
+      .then(firstFiscalYearResponse => {
+        firstFiscalYear.id = firstFiscalYearResponse.id;
+        defaultLedger.fiscalYearOneId = firstFiscalYear.id;
+        Ledgers.createViaApi(defaultLedger)
+          .then(ledgerResponse => {
+            defaultLedger.id = ledgerResponse.id;
+            firstFund.ledgerId = defaultLedger.id;
+            secondFund.ledgerId = defaultLedger.id;
+
+            Funds.createViaApi(firstFund)
+              .then(fundResponse => {
+                firstFund.id = fundResponse.fund.id;
+
+                cy.loginAsAdmin({ path:TopMenu.fundPath, waiter: Funds.waitLoading });
+                FinanceHelp.searchByName(firstFund.name);
+                FinanceHelp.selectFromResultsList();
+                Funds.addBudget(allocatedQuantity);
+              });
+
+            Funds.createViaApi(secondFund)
+              .then(secondFundResponse => {
+                secondFund.id = secondFundResponse.fund.id;
+
+                cy.visit(TopMenu.fundPath);
+                FinanceHelp.searchByName(secondFund.name);
+                FinanceHelp.selectFromResultsList();
+                Funds.addBudget(allocatedQuantity);
+              });
+          });
+      });
+      // Create second Fiscal Year for Rollover
+      FiscalYears.createViaApi(secondFiscalYear)
+      .then(secondFiscalYearResponse => {
+        secondFiscalYear.id = secondFiscalYearResponse.id;
+      });
+      
+      // Prepare 2 Open Orders for Rollover
+      Organizations.createOrganizationViaApi(organization)
+      .then(response => {
```

Comment:
Probably it will be better if you rename 'response'

Reply:
Fixed